x-pack/plugins/aiops/server/routes/explain_log_rate_spikes.ts

```diff
@@ -129,9 +130,13 @@ export const defineExplainLogRateSpikesRoute = (
       }
 
       function end() {
-        isRunning = false;
-        logDebugMessage('Ending analysis.');
-        streamEnd();
+        if (isRunning) {
+          isRunning = false;
+          logDebugMessage('Ending analysis.');
+          streamEnd();
+        } else {
+          logDebugMessage('Stale end analysis call.');
```

Comment:
This debug message `Stale end analysis call.` isn't immediately clear what it means (took me a bit to realize it meant the analysis already ended before end() is called). Suggestion to reword it?

Reply:
Tweaked in [832c39e](https://github.com/elastic/kibana/pull/143683/commits/832c39e23abc39fad877c1e7f23cc60e14f143ab).